### ISSUE TITLE
Words disappearing on wrapped band with newline characters

### ISSUE THREAD

**Issue by gillsonkell** [NathanaelA/fluentreports#135]
When I run this:
```
report.band([
  {
    data: 'Oklahoma Tax Commission\nLine 2',
    width: 100
  }
], {
  wrap: true
});
```
I expect the output to look something like this:
```
Oklahoma Tax
Commission
Line 2
```
but instead it looks like this:
```
Commission
Line 2
```

**Comment by jetalictic** [comment]
'\n' in after Oklahoma tax?

**Comment by NathanaelA** [comment]
This is a bug in the version of the underlying PDFKit -- How critical is this fix; it might be fixed in a later version of PDFKit, but the latest version changes a lot of internal and I haven't wanted to spend the time to update to it with a really good reason.

**Comment by gillsonkell** [comment]
Not critical, just noticed it so I thought I'd throw it out there!

**Comment by NathanaelA** [comment]
Fixed in v1.3.0
